### PR TITLE
[XAI] Use explainable_ai_sdk for tf2 tabular notebooks.

### PR DESCRIPTION
**Pull request by frederick0329**
Leverage [explainable_ai_sdk](https://github.com/GoogleCloudPlatform/explainable_ai_sdk) to improve usability.
The following is improved.

- Metadata creation.

- Making explanation request.

- Visualize response.

### CODE REVIEW

File: notebooks/samples/explanations/tf2/ai-explanations-tabular.ipynb [hunk 1]
```diff
@@ -1,1483 +1,1432 @@
 {
```
Comment (thedriftofwords) [inline notebooks/samples/explanations/tf2/ai-explanations-tabular.ipynb:1]: <p>Replace first paragraph in this section with:</p><p> </p><p>In order to deploy this model to AI Explanations, you need to create an explanation_metadata.json file with information about your model inputs, outputs, and baseline. You can use the&nbsp;<a href="https://pypi.org/project/explainable-ai-sdk/" target="_blank">Explainable AI SDK</a>&nbsp;to generate most of the fields.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (thedriftofwords) [inline notebooks/samples/explanations/tf2/ai-explanations-tabular.ipynb:1]: <p>Change the sentence to:</p><p> </p><p>You can use the Explainable AI SDK to send explanation requests to your deployed model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Comment (frederick0329) [reply notebooks/samples/explanations/tf2/ai-explanations-tabular.ipynb:1]: <p>Done.</p>

**Comment by thedriftofwords** [comment]
The changes work locally for Colab, but would still need some further debugging to pass the "notebook run" test. Merging now, because that isn't launch-blocking.

**Pull request merged**